env/src/env.yml

```diff
@@ -0,0 +1,24 @@
+name: echo
+version: "0.0.0"
+author: Federico Ponzi <federico.ponzi92@gmail.com>
+about: "
+     env [OPTION]... [-] [NAME=VALUE]... [COMMAND [ARG]...]\n\n
+    Set each NAME to VALUE in the environment and run COMMAND.:\n\n"
```

Comment:
About is not usage, put a text describing the utility, like 

```
"Set environment and execute command, or print environment\n\n
The env utility executes another [COMMAND] after modifying the environment as specified on the command line. Each [NAME=VALUE] option specifies the setting of an environment variable, name, with a value of value.  All such environment variables are set before the [COMMAND] is executed."
```
clap generates a good usage by default, just use if it's way worse than expected